packages/spektate/src/repository/IAzureDevOpsRepo.ts

```diff
@@ -114,16 +114,16 @@ export const getManifestSyncState = async (
               accessToken
             );
 
-            if (syncStatus != null) {
+            if (syncStatus != null && syncStatus.data && syncStatus.data.name) {
```

Comment:
Same for this code block
<img width="377" alt="image" src="https://user-images.githubusercontent.com/3119513/79785087-26a3f100-82f8-11ea-9f39-022bc12a1371.png">